core-rust/state-manager/src/protocol/protocol_configs/dumunet_protocol_config.rs

```diff
@@ -44,5 +44,8 @@ pub fn dumunet_protocol_config() -> ProtocolConfig {
                 },
             ],
         },
+        ProtocolVersionName::cuttlefish_part2() => EnactImmediatelyAfterEndOfProtocolUpdate {
```

Comment:
This needs to be `EnactAtStartOfEpochIfValidatorsReady` as well to avoid wiping dumunet (same as stokenet)